e2e/process-services-cloud/task-list-cloud-action-menu.e2e.ts

```diff
@@ -0,0 +1,123 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser } from 'protractor';
+import { GroupIdentityService, IdentityService, LoginSSOPage, QueryService, SettingsPage, TasksService } from '@alfresco/adf-testing';
+import { AppListCloudPage } from '@alfresco/adf-testing';
+import { NavigationBarPage } from '../pages/adf/navigationBarPage';
+import { TasksCloudDemoPage } from '../pages/adf/demo-shell/process-services/tasksCloudDemoPage';
+
+import { ProcessDefinitionsService, ApiService } from '@alfresco/adf-testing';
+import { ProcessInstancesService } from '@alfresco/adf-testing';
+import resources = require('../util/resources');
+
+describe('Process list cloud', () => {
+
+    describe('Process List - Custom Action Menu', () => {
+        const loginSSOPage = new LoginSSOPage();
+        const navigationBarPage = new NavigationBarPage();
+        const appListCloudComponent = new AppListCloudPage();
+        const tasksCloudDemoPage = new TasksCloudDemoPage();
+        const settingsPage = new SettingsPage();
+
+        let processDefinitionService: ProcessDefinitionsService;
+        let processInstancesService: ProcessInstancesService;
+        let identityService: IdentityService;
+        let groupIdentityService: GroupIdentityService;
+        let queryService: QueryService;
+        let tasksService: TasksService;
+        let testUser, groupInfo, editProcess, deleteProcess, editTask, deleteTask;
+
+        const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP.name;
```

Comment:
can we use the same strategy as for the other parameters?
Basically add the resources as part of protractor params and use it as 
`browser.params. resources`

Reply:
I will raise a different issue and refactor all the e2e